storage/kvstore/conf/kv_config.cpp

```diff
@@ -15,40 +15,40 @@
  */
 
 #include "kv_config.h"
-#include "features/storage/kvstore/include/KVStore.h"
-#include "features/storage/kvstore/kv_map/KVMap.h"
-#include "features/storage/blockdevice/BlockDevice.h"
-#include "features/storage/filesystem/FileSystem.h"
-#include "features/storage/kvstore/filesystemstore/FileSystemStore.h"
-#include "features/storage/blockdevice/SlicingBlockDevice.h"
-#include "features/storage/filesystem/fat/FATFileSystem.h"
-#include "features/storage/filesystem/littlefs/LittleFileSystem.h"
-#include "features/storage/kvstore/tdbstore/TDBStore.h"
+#include "storage/kvstore/include/KVStore.h"
+#include "storage/kvstore/kv_map/KVMap.h"
+#include "blockdevice/BlockDevice.h"
+#include "storage/filesystem/FileSystem.h"
+#include "storage/kvstore/filesystemstore/FileSystemStore.h"
+#include "blockdevice/SlicingBlockDevice.h"
+#include "storage/filesystem/fat/FATFileSystem.h"
+#include "storage/filesystem/littlefs/LittleFileSystem.h"
+#include "storage/kvstore/tdbstore/TDBStore.h"
 #include "mbed_error.h"
 #include "drivers/FlashIAP.h"
-#include "features/storage/blockdevice/FlashSimBlockDevice.h"
+#include "blockdevice/FlashSimBlockDevice.h"
 #include "mbed_trace.h"
-#include "features/storage/kvstore/securestore/SecureStore.h"
+#include "storage/kvstore/securestore/SecureStore.h"
 #define TRACE_GROUP "KVCFG"
 
 #if COMPONENT_FLASHIAP
-#include "components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h"
+#include "storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h"
 #endif
 
 #if COMPONENT_QSPIF
-#include "components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.h"
+#include "storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.h"
 #endif
```

Comment:
Just realised we haven't refactored the include paths for `COMPONENT_*/*BlockDevice.h` - they are public APIs but don't follow the "include" proposal in this PR.

Shall we use the same path name `**/include/blockdevice/**` inside each of them, so we would have a consistent
```
#include "blockdevice/FlashIAPBlockDevice.h"
#include "blockdevice/QSPIFBlockDevice.h"
...
```
@rajkan01 @evedon ?

Reply:
Initial blockdevice directory restructure proposal, just to move the source COMPONENT_xxx directly into storage/blockdevice and no refactoring. if we would like to changes on COMPONENT_XXX then we will do it incremental changes

Reply:
I agree that we should do in a subsequent PR.